src/cljs/hello_world/web_socket.cljs

```diff
@@ -50,10 +50,10 @@
 
       :aikakone/game-start (do
                              (swap! util/game-state assoc :sprites-state event-data)
-                             (game/start-game! {:send-sprites-state-fn!   send-sprites-state!
-                                                :send-puzzle-complete-fn! send-puzzle-complete!
-                                                :send-start-timer-fn!     send-start-timer!
-                                                :send-reset-fn!           send-reset!}))
+                             (game/create-puzzle-board {:send-sprites-state-fn!   send-sprites-state!
+                                                        :send-puzzle-complete-fn! send-puzzle-complete!
+                                                        :send-start-timer-fn!     send-start-timer!})
+                             (js/setTimeout send-sprites-state! 300))
```

Comment:
Why do we need this (js/setTimeout send-sprites-state! 300) code here? :) 
There is a same code also in the game.cljs, when we create the play button.

Reply:
Ah! It seems that it is moved from the make-play-button fn. This setTimeout fn is needed for sending a sprite-state after randomising puzzle pieces right? :) Whenever the puzzle is finished, its game-state is going be to be empty map {}, and puzzle pieces will be randomly spreaded, I guess, so the new sprite-state needs to be sent to the server to share(sync) the new game state with other client.

`(let [initial-sprites-state (:sprites-state @util/game-state)]
    (if (not (empty? initial-sprites-state))
      (util/synchronize-puzzle-board initial-sprites-state)
      ;It make puzzle pieces randomly flipped,
      ;if it is the initial puzzle creation.
      (do (randomize-puzzle-pieces)
          (send-start-timer-fn!))))
  (util/display-play-time!))`

Reply:
yeah just sending the state of puzzle board after `create-puzzle-board`